formula/src/test/java/com/instacart/formula/FormulaRuntimeTest.kt

```diff
@@ -1078,27 +1124,36 @@ class FormulaRuntimeTest(val runtime: TestableRuntime, val name: String) {
 
     @Test
     fun `initialize 100 levels nested formula`() {
+        val inspector = CountingInspector()
         val formula = ExtremelyNestedFormula.nested(100)
-        runtime.test(formula, Unit).output {
+        runtime.test(formula, Unit, inspector).output {
             assertThat(this).isEqualTo(100)
         }
+        inspector.assertRunCount(1)
+        inspector.assertEvaluationCount(5150)
```

Comment:
Quite a number of evaluations (in https://github.com/instacart/formula/pull/301 this is down to 200)